LayoutTests/accessibility/full-size-kana-untransformed.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="../resources/accessibility-helper.js"></script>
+<script src="../resources/js-test.js"></script>
+<meta charset=utf-8>
+</head>
+<body id="body">
```

Comment:
Is this `id` necessary?

Reply:
Nope (left over from a copy-and-paste). I'll remove it in the next revision.